frontend/src/cloud-hosting/useCloudHosting.ts

```
@@ -294,6 +295,9 @@ export function useCloudHosting() {
   const posthogStatusSubscribersRef = useRef<
     Set<(event: DashboardPosthogConnectionStatus) => void>
   >(new Set());
+  const apifyStatusSubscribersRef = useRef<
+    Set<(event: DashboardApifyConnectionStatus) => void>
```

ESLint

[error] Replace `⏎····Set<(event:·DashboardApifyConnectionStatus)·=>·void>` with `Set<(event:·DashboardApifyConnectionStatus)·=>·void>>(`

(prettier/prettier)